desktop/package/linux/Haveno.AppDir/Haveno.desktop

```
@@ -0,0 +1,15 @@
+# If you want Haveno to appear in a Linux app launcher ("start menu"), install this by doing:
+# sudo desktop-file-install electrum-ltc.desktop
+
+[Desktop Entry]
+Comment=A decentralized monero exchange network.
+Exec=sh -c "PATH=\"\\$HOME/.local/bin:\\$PATH\"; bin/Haveno %u"
+GenericName[en_US]=Monero Exchange
+GenericName=Monero Exchange
+Icon=haveno
+Categories=Office;Finance;Java;
```

There's a P2P category that would be nice to add:
```suggestion
Categories=Office;Finance;Java;P2P;
```

---

Fixed.